package.json

```diff
@@ -14,11 +14,15 @@
     "@neondatabase/serverless": "^0.9.5",
     "@vercel/speed-insights": "^1.0.12",
     "bcryptjs": "^2.4.3",
+    "dotenv": "^16.4.5",
     "drizzle-kit": "^0.24.2",
     "drizzle-orm": "^0.33.0",
-    "next": "14.2.5",
+    "framer-motion": "^11.11.10",
+    "gsap": "^3.12.5",
+    "next": "^14.2.16",
     "next-auth": "^5.0.0-beta.22",
     "nodemailer": "^6.9.15",
+    "pg": "^8.13.0",
```

Comment:
remove it

Reply:
> remove it

i got errors while setting up the project , so i did it

Reply:
> > remove it
> 
> i got errors while setting up the project , so i did it

Error: No database connection string was provided to `neon()`. Perhaps an environment variable has not been set?

This error happened while generating the page. Any console logs will be displayed in the terminal window.
Source
src/lib/drizzle.ts (8:19) @ connectionString

   6 | const connectionString = process.env.DATABASE_URL!;
   7 |
>  8 | const pool = neon(connectionString)
     |                   ^
   9 |
  10 | export const db = drizzle(pool, { schema })
Call Stack

Reply:
> > remove it
> 
> i got errors while setting up the project , so i did it

Error: No database connection string was provided to `neon()`. Perhaps an environment variable has not been set?

This error happened while generating the page. Any console logs will be displayed in the terminal window.
Source
src/lib/drizzle.ts (8:19) @ connectionString

   6 | const connectionString = process.env.DATABASE_URL!;
   7 |
>  8 | const pool = neon(connectionString)
     |                   ^
   9 |
  10 | export const db = drizzle(pool, { schema })
Call Stack